pkg/polaris/scanner.go

```diff
@@ -28,32 +29,29 @@ const (
 	configVolume         = "config"
 )
 
-type Config interface {
-	GetPolarisImageRef() (string, error)
-}
-
+// TODO Move the Scanner struct to configauditreport package. It's generic and can be used with similar tools to Polaris.
```

Comment:
I will do that in a separate PR. Otherwise reviewing the changes would be difficult.